packages/code-du-travail-frontend/src/data/themes-front.json

```diff
@@ -0,0 +1 @@
+../../../code-du-travail-data/dataset/themes-front.json
```

Comment:
Hum... 
That will work but why not using the file from the node_modules ?

```
@cdt/data/dataset/themes-front.json
```

Reply:
didn't know we can do that since I touhg data doesn't export any thing but It works 
thanks !